azuredeploy.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
this is useful for  deploying from github the same image. We can remove azuredeploy.json